agenta-cli/agenta/sdk/agenta_decorator.py

```diff
@@ -374,3 +376,10 @@ def find_in_schema(schema: dict, param_name: str, xparam: str):
         if isinstance(param_val, TextParam):
             subschema = find_in_schema(schema_to_override, param_name, "text")
             subschema["default"] = param_val
+        if (
+            isinstance(param_val, inspect.Parameter)
+            and param_val.annotation is MessagesParam
+        ):
+            subschema = find_in_schema(schema_to_override, param_name, "messages")
+            del subschema["items"]
```

Comment:
why do we have this line?

Reply:
I was deleting a property that we won't be using, turns out the design isn't the right now. I've gone ahead to remove it. Thank you very much!